ex1-tema2.js

```diff
@@ -7,4 +7,22 @@ function even_numbers (n){
 }
 even_numbers(10);
 
+//Tipăriți toate numerele impare  de la 0 la 10
 
+//Method 1
+function even_numbers (n){
+    for (let i=0; i<=n; i++){
+        if (i%2 !==0) {
+            console.log(i);
+        }    
+    }
+}
+even_numbers(10);
```

Comment:
(optional)
O idee de imbunatatire poate fi ca aceasta functie sa returneze o lista intreaga iar afisarea sa se intample in afara functiei.